src/panels/element/PackageDivider.tsx

```diff
@@ -23,7 +23,8 @@ export default class PackageDivider extends React.Component<Props, State> {
 	}
 
 	render() {
-		return (<div className={"packageDivider"}
+		return (<div className={"packageDivider" + (this.props.visible ? "" : " closed") +
```

Comment:
I recommend taking a look at this package that can help with classes definitions: https://www.npmjs.com/package/classnames

Reply:
I tried the package and liked it, but I won't put it in this PR as that would produce a merge conflict. Will implement next time, promise!